.github/workflows/extensions.yml

```diff
@@ -0,0 +1,170 @@
+name: Extension Workflow
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  # Detects file changes in extension directories that utilize the `simple-extensions`
+  # job below
+  simple-extension-changes:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      # Expose matched filters as 'changes' output variable
+      changes: ${{ steps.changes.outputs.changes }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          # Adding a new extension that has a directory that can be TARed?
+          # Add a new line here with the name of the extension and directory path
+          # Be sure the extension name and directory have the same name
+          # e.g. `extension-name: extensions/extension-name/**`
+          filters: |
+            reaper: extensions/reaper/**
+            integration-session-manager: extensions/integration-session-manager/**
```

Comment:
After a conversation with @cgraham-rs and the recommendation from @edavidaja I _think_ we can get the simple extension flow to require no change from the extension authors. It will require some additional investigation, but the idea would be to find every `extension/**` directory with a signifier in the manifest that it is a simple extension, and see if it has changes.

Reply:
A quick look into this I cannot find a way to do both matrix directories and see which have changed without some inputs. There is probably a way to do this, but it is more complex than I expected. We can tackle this if the feedback from the CONTRIBUTING documentation is that it is too complex.